l10n_be_national_number/migrations/16.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    for id_numbers in env["res.partner"].search([]).mapped("id_numbers"):
```

Comment:
```suggestion
    for id_numbers in (
        env["res.partner"]
        .with_context(active_test=False)
        .search([])
        .mapped("id_numbers")
    ):
```
`.with_context(active_test=False)` is needed to also convert id numbers of inactive partners.